dataverse_download.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/local/bin/python
+# Script to download a dataset from a Dataverse (https://dataverse.org/)
+#' @author
+#' John Bradley: initial code
+#' Thibault Tabarin: small modification
```

Comment:
For all of these author comments could we separate out the work into commits? That way you could tell which parts are yours and mine. Especially if there is a bug in the part I wrote :)
If a commit message starting with `Co-authored-by:` it will display the coauthors in the UI. Details:
https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

Reply:
I get the idea and I agree but how to do it exactly... and yes I read the doc.... but does it mean I should reverse the commit and restart at the beginning with co-authorship commit?

Reply:
if you don't mind I will do next time... I seems too complicated to fix it from here unless you know how to do it